cli/Dockerfile-bin

```diff
@@ -8,18 +12,22 @@ COPY controller/gen controller/gen
 COPY pkg pkg
 RUN mkdir -p /out
 
+# Packr needs to build in the directory that the Box is relative to
+WORKDIR /go/src/github.com/linkerd/linkerd2/cli
+
 # Cache builds without version info
-RUN CGO_ENABLED=0 GOOS=darwin  go build -o /out/linkerd-darwin  -ldflags "-s -w" ./cli
-RUN CGO_ENABLED=0 GOOS=linux   go build -o /out/linkerd-linux   -ldflags "-s -w" ./cli
-RUN CGO_ENABLED=0 GOOS=windows go build -o /out/linkerd-windows -ldflags "-s -w" ./cli
+RUN CGO_ENABLED=0 GOOS=linux   packr2 build -o /out/linkerd-linux   -ldflags "-s -w"
+RUN CGO_ENABLED=0 GOOS=windows packr2 build -o /out/linkerd-windows -ldflags "-s -w"
+RUN CGO_ENABLED=0 GOOS=darwin  packr2 build -o /out/linkerd-darwin  -ldflags "-s -w"
```

Comment:
What's the motivation for reordering the architectures here? My preference is to keep them the way they were, to match the ordering of lines 25-27 in this file.

Reply:
Oh I did not mean to do this. It looks like I shifted the `darwin` line down two times after confusing it for the hotkey that would add add two additional cursors (to make the same edits on all three lines).